uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-month.component.js

```diff
@@ -241,25 +241,26 @@ define([
     }
 
     /**
-     * Returns whether a leaveRequest is pending approval
+     * Decides whether sent date is a public holiday
      *
-     * @param  {object} leaveRequest
+     * @param  {string} date
      * @return {boolean}
      */
-    function isPendingApproval (leaveRequest) {
-      var status = vm.supportData.leaveRequestStatuses[leaveRequest.status_id];
-
-      return status.name === sharedSettings.statusNames.awaitingApproval;
+    function isPublicHoliday (date) {
+      return !!vm.supportData.publicHolidays[dateObjectWithFormat(date).valueOf()];
     }
 
     /**
-     * Decides whether sent date is a public holiday
+     * Returns whether a leaveRequest is pending approval or more information requested
      *
-     * @param  {string} date
+     * @param  {object} leaveRequest
      * @return {boolean}
      */
-    function isPublicHoliday (date) {
-      return !!vm.supportData.publicHolidays[dateObjectWithFormat(date).valueOf()];
+    function isRequested (leaveRequest) {
+      var status = vm.supportData.leaveRequestStatuses[leaveRequest.status_id];
+
+      return status.name === sharedSettings.statusNames.awaitingApproval ||
```

Comment:
How about this?
```js
return _.include(vm.supportData.leaveRequestStatuses[leaveRequest.status_id].name,
  [sharedSettings.statusNames.awaitingApproval,
    sharedSettings.statusNames.moreInformationRequired];
```

Reply:
I had to use `contains` as `include`is not working. And kept the variable separate for more readability.